app/main.py

```
@@ -1,18 +1,24 @@
 @click.command()
 @click.option(
     "--transport",
-    type=click.Choice(["stdio", "sse"]),
+    type=click.Choice(["stdio", "sse", "streamable-http"]),
     default="stdio",
     help="Transport type",
 )
 def main(transport: str):
-    app.run(transport=transport)
+    if transport == "streamable-http":
+        uvicorn.run(create_http_app(), host="0.0.0.0", port=int(os.environ.get("PORT", "8000")))
```

Ruff (0.15.21)

[error] Possible binding to all interfaces

(S104)